ebean-core/src/main/java/io/ebeaninternal/server/type/ScalarTypeJsonObjectMapper.java

```diff
@@ -149,11 +156,23 @@ public boolean isMutable() {
     }
 
     /**
-     * Return true if the value should be considered dirty (and included in an update).
+     * Return true if the value should be considered dirty (and included in an
+     * update).
      */
     @Override
     public boolean isDirty(Object value) {
-      return dirtyHandler.isDirty(value);
```

Comment:
The dirtyHandler is obsolete now, 
we could also check, if we change the interface of the 
```java
DirtyHandler {
    boolean isDirty(Object value);

    void track(Object value, String json);
}
```
and  move the functionality to the dirtyHandler.

I am also thinking, if we should extend `ModifyAwareType` or an other interface with `set/getTrackingJson()`
Then we can use the object itself to store the "tracking value" instead of the weakReference-construct